repos/system_upgrade/common/actors/setuptargetrepos/libraries/setuptargetrepos_repomap.py

```diff
@@ -0,0 +1,296 @@
+from leapp.libraries.common.config.version import get_source_major_version, get_target_major_version
+from leapp.libraries.stdlib import api
+
+
+DEFAULT_PESID = {
+    '7': 'rhel7-base',
+    '8': 'rhel8-BaseOS',
+    '9': 'rhel9-BaseOS'
+}
+
+
+def _get_channel_prio(pesid_repo):
+    priorities = {
+        'beta': 0,
+        'ga': 1,
+    }
+    return priorities.get(pesid_repo.channel, 10)
```

Comment:
Do we have this default 10 value described somewhere? Do I understand it correctly that it means the highest EUS/E4S channel?

Reply:
No. It's private library so we just set 10 in case we would like to put another priority level between 'ga' and the top one. Yes, that's the highest prio.